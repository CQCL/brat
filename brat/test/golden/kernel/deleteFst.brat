deleteFst :: { Qubit, Qubit -o Qubit }
deleteFst = { q0, q1 => q1 }
