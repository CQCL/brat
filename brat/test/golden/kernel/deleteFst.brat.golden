Error in test/golden/kernel/deleteFst.brat@2:13-29
deleteFst = { q0, q1 => q1 }
            ^^^^^^^^^^^^^^^^

  Type error: Variable(s) q0 haven't been used

